dpgen/simplify/simplify.py

```diff
@@ -411,6 +411,10 @@ def make_fp(iter_index, jdata, mdata):
     else:
         make_fp_configs(iter_index, jdata)
         make_fp_calculation(iter_index, jdata)
+    # Copy user defined forward_files
+    iter_name = make_iter_name(iter_index)
+    work_path = os.path.join(iter_name, fp_name)
+    symlink_user_forward_files(mdata=mdata, task_type="fp", work_path=work_path)
```

Comment:
These lines should be executed only when `labeled` is `False`.

Reply:
Also, do we already have corresponding parameters?

Reply:
> Also, do we already have corresponding parameters?

It can be set in `machine.json`, just like `run`—tested work.